components/Header.tsx

```diff
@@ -5,40 +5,51 @@ import { useRouter } from 'next/router';
 import { IoLogoGithub } from 'react-icons/io';
 import { BsPeopleFill } from 'react-icons/bs';
 
-import Search from './Search';
+import Search, { SearchProps } from './Search';
 
-function Header() {
+
+const SearchBar = (props: SearchProps) => {
   const router = useRouter();
 
+  return (router.pathname === '/repos/[language]' && (
+      <Search {...props} />
+    )
+  )
+}
+
+function Header() {
   return (
-    <div className="border-b-[0.5px] border-dashed border-b-2023-manga-3 flex justify-center mb-5">
-      <div className="justify-between px-2 mb-2 ml-1.5 w-3/4 shadow-lg navbar rounded-box">
-        <Link href="/">
-          <img src="/hacktoberfest.svg" alt="Hacktoberfest" />
-        </Link>
-        {router.pathname === '/repos/[language]' && (
-          <div className="justify-center flex-1">
-            <Search />
-          </div>
-        )}
-        <div className="flex-none">
-          <Link
-            href="/contributors"
-            className="btn btn-square btn-ghost umami--click--contributors-button"
-          >
-            <BsPeopleFill size="1.5rem" color='white' title="Contributors" />
+    <header className="border-b-[0.5px] border-dashed border-b-2023-manga-3 mb-5">
+      <div className="container mx-auto px-4 py-2">
+        <div className="justify-between shadow-lg navbar">
+          <Link href="/">
+            <img src="/hacktoberfest.svg" alt="Hacktoberfest" className="h-12 w-auto sm:h-auto" />
           </Link>
-          <a
-            href="https://github.com/max-programming/hacktoberfest-projects"
-            target="_blank"
-            rel="noreferrer"
-            className="btn btn-square btn-ghost umami--click--github-button"
-          >
-            <IoLogoGithub size="1.5rem" color='white' title="GitHub" />
-          </a>
-        </div>
+
+          <SearchBar searchBarWrapperStyles="hidden sm:inline-flex flex-1 max-w-md px-6" />
+
+          <div className="flex-none">
+            <Link
+              href="/contributors"
+              className="btn btn-square btn-ghost umami--click--contributors-button"
+            >
+              <BsPeopleFill size="1.5rem" color='white' title="Contributors" />
+            </Link>
+
+            <a
+              href="https://github.com/max-programming/hacktoberfest-projects"
+              target="_blank"
+              rel="noreferrer"
+              className="btn btn-square btn-ghost umami--click--github-button"
+            >
+              <IoLogoGithub size="1.5rem" title="GitHub" />
```

Comment:
Add `color='white'` to `IoLogoGithub`

Reply:
![image](https://github.com/max-programming/hacktoberfest-projects/assets/61869195/ac1a975e-79e0-4ede-862b-eaa245526ae7)

@prithvi2k2  I can see its already white, can you share the screenshot how its looking for you?

Reply:
That is a different deployment by hacktoberfest, check out original deployment @ https://finder.usmans.me/

Reply:
Wait, even that is gone now 
![image](https://github.com/max-programming/hacktoberfest-projects/assets/72189314/12116750-c9af-4542-b70c-7f2e9380a50f)

Reply:
![image](https://github.com/max-programming/hacktoberfest-projects/assets/61869195/a1fb8630-06d7-45c9-9442-b060780606f8)

Its already white on it as well

Reply:
@NishidhJain Maybe it's your browser messing up with colors, try it in dark mode or mobile - preferably incognito

Reply:
> ![image](https://user-images.githubusercontent.com/61869195/272169699-a1fb8630-06d7-45c9-9442-b060780606f8.png)
> 
> Its already white on it as well

I tried with different browsers and on mobile too, did you check it with other browsers or platforms? It's still black for me
![image](https://github.com/max-programming/hacktoberfest-projects/assets/72189314/3bf80cf1-1abd-4a76-a299-fb6b617d5eb1)

Reply:
I just checked in incognito mode and on mobile device its showing in white. Also, I have dark mode turned on. @max-programming Can you please confirm how its looking on your device?

Reply:
> I just checked in incognito mode and on mobile device its showing in white. Also, I have dark mode turned on. @max-programming Can you please confirm how its looking on your device?

oh well. that is the cause of issue then, dark mode - as far as I've seen, it messed up many elements for me many times on various pages... I'm talking about 3rd party extensions for dark mode and Forced Dark Modes in browser. Can you turn it off and check?